spec/system/provider_interface/provider_content_spec.rb

```diff
@@ -68,12 +68,12 @@ def then_i_can_see_the_complaints_page
     expect(page).to have_content('Make a complaint about this service')
   end
 
-  def when_i_click_on_the_privacy_policy
-    within('.govuk-footer') { click_link t('layout.support_links.privacy_policy') }
+  def when_i_click_on_privacy
+    within('.govuk-footer') { click_link t('layout.support_links.privacy') }
   end
 
-  def then_i_can_see_the_privacy_policy
-    expect(page).to have_content(t('page_titles.privacy_policy'))
+  def then_i_can_see_the_privacy_notices
+    expect(page).to have_content(t('page_titles.privacy_notices'))
```

Comment:
maybe we can step into each policy page, what do you think?

Reply:
Sure, sounds like a good idea